src/main/java/Task.java

```diff
@@ -0,0 +1,35 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task() {
+        this.description = "";
+        this.isDone = false;
+    }
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public boolean getIsDone() {
+        return isDone;
+    }
+
+    public void setIsDone(boolean isDone) {
+        this.isDone = isDone;
+    }
+
+    @Override
+    public String toString() {
+        return "[" + (isDone ? Duke.tick : Duke.cross) + "] " + description;
```

Comment:
To increase the code readability, you can replace the (... ? ... : ...) statement with an if-else statement. But line 33 still seems quite okay to me.

Reply:
Noted, thanks!